app/request_lookup.py

```diff
@@ -0,0 +1,34 @@
+import json
+import requests
+from app import settings
+
+request_timeout = 30
+
+def query_country_index(search_value):
+    return get_results('country', search_value)
+
+def query_occupation_index(search_value):
+    return get_results('occupation', search_value)
+
+def query_industry_index(search_value):
+    return get_results('industry', search_value)
+
+
+def get_results(type, search_value):
+    request_string = settings.LOOKUP_URL + type + '/_search?q=' + search_value
```

Comment:
I If you use `os.path.join(settings.LOOKUP_URL, type, '_search?q=' + search_value)`
then this will handle the potential absence of a trailing slash on `LOOKUP_URL`